packages/hash/api/src/graphql/typeDefs/knowledge/page.typedef.ts

```diff
@@ -254,7 +272,57 @@ export const knowledgePageTypedef = gql`
     placeholders: [UpdateKnowledgePageContentsResultPlaceholder!]!
   }
 
+  input KnowledgePageCreationData {
+    """
+    The page title.
+    """
+    title: String!
+    """
+    The index of the page that is before the current.
```

Comment:
Similar question about fractional index

Reply:
Addressed in https://github.com/hashintel/hash/pull/1157/commits/5c71f70bc5f067f1883fabb47fa7ccddc373e8f3